diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigTest.kt

```diff
@@ -26,8 +26,8 @@ class RulesConfigJsonTest {
             val ruleJson = jacksonMapper.writeValueAsString(jsonCodeSnippet)
             Assertions.assertTrue(foundRule != null) {
                 """
-                   Cannot find warning ${rule.ruleName()} in rules-config.json.
-                   You can fix it by adding the following code below to rules-config.json:
+                   Cannot find warning ${rule.ruleName()} in diktat-analysis.yml.
+                   You can fix it by adding the following code below to diktat-analysis.yml:
                    $ruleJson
```

Comment:
This test will still suggest json snippet, but ideally it should suggest yml

Reply:
Why did you resolve comment and not address it's message? Please either reply or edit code.

Reply:
OK, turns out it already suggests yml